pom.xml

```diff
@@ -36,7 +36,9 @@ SOFTWARE.
   <name>polystat</name>
   <description>Polyglot Static Analyzer</description>
   <properties>
-    <eolang.version>0.1.30</eolang.version>
+    <eolang.version>0.5.0</eolang.version>
+    <odin.version>0.1.1</odin.version>
+    <reactivstream.version>1.0.3</reactivstream.version>
```

Comment:
@Sitiritis typo? what this is for?

Reply:
Yes, forgot to remove the redundant dependency